.gitmessage

```diff
@@ -0,0 +1,11 @@
+build(scope): Changes that affect the build system or external dependencies
```

Comment:
Run `git config --global commit.template /path/to/.gitmessage` to activate this template.